### DISCUSSION TITLE
MFA - nawet gdy użytkownik przy konfiguracji MFA użycje SMS - domyślną metodą logowania: Telefon - połączenie

### DISCUSSION THREAD

**Discussion by MariuszFerdyn** [AzureCommunityPL/group#252] [Pomoc]
Krok po kroku:

1. Włączam usługę MFA dla użytkownika
2. Użytkownik zamiast aplikacji Authenticator wybiera inną metodę, a następnie Telefon, wiadomość tekstową
3. Wpisuje otrzymaną wiadomość tekstową (SMS)
4. użytkownik widzi domyślną metodę logowania: będzie Telefon - połączenie

Powstaje pytanie dlaczego użytkownik wybiera SMS (wiadomość tekstowa), ale domyślnie dla MFA będzie to połączenie? Wiem, że użytkownik może to później zmienić, ale... ile to tłumaczenia.... Ktoś to ogarniał?

Kojarzy ktoś czy mogę zmienić dla wszystkich nowych użytkowników aby MFA domyślnie korzystało z SMS, a nie z połączenia telefonicznego? Jakieś inne hinty?

**Comment by maciejfrydrych** [comment]
Może ten sposób pomoże. Kiedyś z niego korzystałem, nie wiem czy notatki mam dobre. 

$User = Get-MSolUser -UserPrincipalName ***@***.***
$User.StrongAuthenticationMethods 

Listuje metody dla MFA. Poniżej to co mam zapisane odnośnie ustawiania MFA. 

$UPN = ***@***.*** "
$MFA = New-Object -TypeName Microsoft.Online.Administration.StrongAuthenticationMethod
$MFA.IsDefault = $true
$MFA.MethodType = "OneWaySMS"
$Phone = New-Object -TypeName Microsoft.Online.Administration.StrongAuthenticationMethod
$Phone.IsDefault = $false
$Phone.MethodType = "TwoWayVoiceMobile"
$MFAMethods = @($SMS, $Phone)
Get-MsolUser -All  | foreach{set-msoluser -UserPrincipalName $_.UserPrincipalName -StrongAuthenticationMethods $ MFAMethods }

Plus art. O raporcie MFA u userów
https://office365itpros.com/2018/11/21/reporting-mfa-enabled-accounts/

Pozdrawiam
From: Mariusz Ferdyn
Sent: niedziela, 21 marca 2021 14:37
To: AzureCommunityPL/group
Cc: Subscribed
Subject: [AzureCommunityPL/group] MFA - nawet gdy użytkownik przy konfiguracji MFA użycje SMS - domyślną metodą logowania: Telefon - połączenie (#252)

Krok po kroku:
1. Włączam usługę MFA dla użytkownika
2. Użytkownik zamiast aplikacji Authenticator wybiera inną metodę, a następnie Telefon, wiadomość tekstową
3. Wpisuje otrzymaną wiadomość tekstową (SMS)
4. użytkownik widzi domyślną metodę logowania: będzie Telefon - połączenie
Powstaje pytanie dlaczego użytkownik wybiera SMS (wiadomość tekstowa), ale domyślnie dla MFA będzie to połączenie? Wiem, że użytkownik może to później zmienić, ale... ile to tłumaczenia.... Ktoś to ogarniał?
Kojarzy ktoś czy mogę zmienić dla wszystkich nowych użytkowników aby MFA domyślnie korzystało z SMS, a nie z połączenia telefonicznego? Jakieś inne hinty?
